src/Templates/ProjectTemplates/CSharp/.NETCore/ConsoleApplication/NuGet.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
```

Comment:
The `Microsoft.NETCore.App` dependency should go "under" the `netcoreapp1.0` framework, as this only applies to that framework.  See https://github.com/dotnet/cli/blob/rel/1.0.0/src/dotnet/commands/dotnet-new/CSharp_Console/project.json.template for the current template.